firebase-database/src/androidTest/java/com/google/firebase/database/DatabaseRegistrarTest.java

```diff
@@ -0,0 +1,28 @@
+package com.google.firebase.database;
+
+import com.google.firebase.platforminfo.UserAgentPublisher;
+import org.junit.Test;
+
+import static com.google.common.truth.Truth.assertThat;
+
+public class DatabaseRegistrarTest {
```

Comment:
you need an explicit `@RunWith`, otherwise tests won't run in bazel.

See: https://github.com/firebase/firebase-android-sdk/pull/223